paket.dependencies

```diff
@@ -45,11 +45,11 @@ nuget Be.Vlaanderen.Basisregisters.Build.Pipeline 6.0.5
 
 nuget Be.Vlaanderen.Basisregisters.Api 18.2.1
 
-nuget Be.Vlaanderen.Basisregisters.GrAr.Common 17.29.0
-nuget Be.Vlaanderen.Basisregisters.GrAr.Edit 17.29.0
-nuget Be.Vlaanderen.Basisregisters.GrAr.Legacy 17.29.0
-nuget Be.Vlaanderen.Basisregisters.GrAr.Provenance 17.29.0
-nuget Be.Vlaanderen.Basisregisters.GrAr.Import 17.29.0
+nuget Be.Vlaanderen.Basisregisters.GrAr.Common 17.32.0
```

Comment:
Shouldn't you also update streetname registry dependency in order that the new event is included in the documentation?